c/le.c

```diff
@@ -266,7 +266,7 @@ void initRLEEnvironment(RLEAnchor *anchor) {
 
   int recoveryRC = recoveryEstablishRouter(RCVR_ROUTER_FLAG_NONE);
   if (recoveryRC != RC_RCV_OK) {
-    printf("le.c: error - recovery router not established\n");
+    printf("le.c: error - recovery router not established\n"); // TODO: Needs message ID
```

Comment:
it seems like this rc should be returned, but isnt. is printf really the best we can do here?

Reply:
I would return some RC and not have an official message here.